crypto/src/mls/session/error.rs

```diff
@@ -31,8 +31,6 @@ pub enum Error {
     IdentityAlreadyPresent,
     #[error("The supplied credential does not match the id or signature schemes provided")]
     WrongCredential,
-    #[error("An EpochObserver has already been registered; reregistration is not possible")]
-    EpochObserverAlreadyExists,
```

Comment:
Do we really want to relax this restriction? It's hard to envison non-buggy cases where the client registers an epoch observer more than once.

Reply:
Right, maybe you're right. I was aligning it with how the `mls_transport` callback.

I'll bring it back.